src/backend/backconfig.c

```diff
@@ -79,7 +80,9 @@ void out_config_init(
         config.objfmt = OBJ_MSCOFF;
     }
     else
-    {   config.exe = EX_NT;
+    {
+        config.exe = EX_NT;
+        config.ehmethod = EH_WIN32;
         config.flags2 |= CFG2seh;       // Win32 eh
```

Comment:
At the moment, it doesn't work if I remove it. The logic that uses these configuration things is convoluted, this is the first step towards straightening it out, which is necessary in order to support dwarf eh.